src/twin/provisioning_config.rs

```diff
@@ -18,11 +18,22 @@ macro_rules! identity_config_file_path {
     }};
 }
 
-macro_rules! device_cert_file_path {
-    () => {{
-        const ENV_FILE_PATH_DEFAULT: &'static str = "/var/lib/aziot/certd/certs/deviceid-*.cer";
-        env::var("DEVICE_CERT_FILE_PATH").unwrap_or(ENV_FILE_PATH_DEFAULT.to_string())
-    }};
+macro_rules! cert_file_path {
+    ($val:expr) => {
+        match $val {
+            true => {
+                const EST_CERT_FILE_PATH_DEFAULT: &'static str =
+                    "/var/lib/aziot/certd/certs/deviceid-*.cer";
+                env::var("EST_CERT_FILE_PATH").unwrap_or(EST_CERT_FILE_PATH_DEFAULT.to_string())
+            }
+            false => {
+                const DEVICE_CERT_FILE_PATH_DEFAULT: &'static str =
+                    "/mnt/cert/priv/device_id_cert.pem";
+                env::var("DEVICE_CERT_FILE_PATH")
+                    .unwrap_or(DEVICE_CERT_FILE_PATH_DEFAULT.to_string())
+            }
+        }
+    };
```

Comment:
We only use this macro once, don't we? Maybe we could move the glob part into the macro, and change it to a function/struct that impls IntoIter<Type=Path>?